src/app/widgets/Rotary/Rotary.js

```diff
@@ -78,7 +78,7 @@ const Rotary = () => {
         },
         jog: (params = {}) => {
             const modal = 'G21';
-            const s = map(params, (value, letter) => ('' + letter.toUpperCase() + value)).join(' ');
+            const s = map(params, (value, letter) => ('' + (letter === 'a' || letter === 'A' ? 'Y' : letter.toUpperCase()) + value)).join(' ');
```

Comment:
Same drill here, change the letter property to the necessary axis letter when calling the function.